### ISSUE TITLE
DNS Record Import `name` does not match TF import

### ISSUE THREAD

**Issue by garrettgalow** [cloudflare/cf-terraforming#23]
When using cf-terraforming on a dns record the record is created as:
`name: "<subdomain>.<apex>"`

For domain mitigateddos.net, for www:
`name: "www.mitigateddos.net"`

On TF import of that resource the name is imported as `www`

This leads to a state conflict after cf-terraforming import state and TF import of the tfstate:
```
-/+ cloudflare_record.mitigateddos_net_www_mitigateddos_net (new resource required)
      id:          "d09d916d059aa9fc8cb54bdd49deea5f" => <computed> (forces new resource)
      created_on:  "2018-04-05T13:43:59.54833Z" => <computed>
      domain:      "mitigateddos.net" => "mitigateddos.net"
      hostname:    "www.mitigateddos.net" => <computed>
      metadata.%:  "3" => <computed>
      modified_on: "2018-04-05T13:43:59.54833Z" => <computed>
      name:        "www" => "www.mitigateddos.net" (forces new resource)
      proxiable:   "true" => <computed>
      proxied:     "true" => "true"
      ttl:         "1" => "1"
      type:        "CNAME" => "CNAME"
      value:       "mitigateddos.net" => "mitigateddos.net"
      zone_id:     "be9827cd1241f1f259c6af218827dc7b" => <computed>
```

cf-terraforming should import the config as the tfstate would expect

**Comment by garrettgalow** [comment]
fixing the name leads to synchronized state between config & state files

**Comment by garrettgalow** [comment]
for non apex records removing the apex as a suffix should be sufficient to fix this. aka `test.www.example.com` removing the suffix of `example.com` leads to `test.www`

only exception known is if the record is only apex then just return the string

**Comment by patryk** [comment]
Fixed by #84.
